config.json

```diff
@@ -1036,6 +1036,15 @@
         "difficulty": 3,
         "topics": [
         ]
+      },
+      {
+        "slug": "rotational-cipher",
+        "name": "Rotational Cipher",
+        "uuid": "21d02535-4ab6-455d-88e1-08ec3b578894",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": []
```

Comment:
I think "strings" and "loops" would be good topics.
Maybe "math", too, because of the modulo arithmetic.

Reply:
Every concept that is added in the "practices" field, makes the syllabus tree a bit bigger. Loops is quite big already, but we could add loops as a prereq and add it to the strings concept, so it is only available if people finished loops?

Reply:
Oh, I didn't realize that. I'll try to remember that.